ledger/participant-integration-api/src/main/resources/db/migration/oracle-appendonly/V8__add_user_managment.sql

```diff
@@ -0,0 +1,19 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+
+CREATE TABLE participant_users (
+    internal_id         NUMBER                GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    user_id             VARCHAR2(4000)         NOT NULL UNIQUE,
+    primary_party       VARCHAR2(4000),
```

Comment:
Parties are implemented as VARCHAR2(1000) elsewhere in the oracle database. Why larger size here? Same applies to the rights table

Reply:
User id VARCHAR2(256)
Parties 1000

Reply:
Afaict parties (party id) are implemented as:
- `VARCHAR2(4000)` in Oracle,
- `TEXT` in Postgres,
-  `TEXT` in H2
which follows from string interning tables.

Oracle:
```
CREATE TABLE string_interning (
    internal_id      NUMBER          PRIMARY KEY NOT NULL,
    external_string  VARCHAR2(4000)
);
```


Postgres:
```
CREATE TABLE string_interning (
    internal_id     INTEGER PRIMARY KEY NOT NULL,
    external_string TEXT
);
```

H2:
```
CREATE TABLE string_interning (
    internal_id integer PRIMARY KEY NOT NULL,
    external_string text
);
```

Reply:
User id VARCHAR2(256)
Parties 512